plan_delegator/src/plan_delegator.cpp

```diff
@@ -250,7 +250,10 @@ namespace plan_delegator
         // set current vehicle state if we have NOT planned any previous trajectories
         if(latest_trajectory_plan.trajectory_points.empty())
         {
-            plan_req->header.stamp = latest_pose_.header.stamp;
+            plan_req->header.stamp = now(); // used to be latest_pose_.header.stamp which was not reliable
```

Comment:
Why was that not reliable? This change creates an inconstancy between the pose in the planning request and the request time.

Reply:
Initially created this change because latest_pose was not being updated in the plugin due to service calls being timed out. It is actually not conclusive if the repeatedly failed service calls are causing the node to freeze or it's because something else is freezing which causing the both tasks to fail. However, decreasing the execution time of the tactical plugin helped significantly reduce the service call failures, which also removed the need for this change. So reverting it back to latest_pose's stamp which keeps the consistency between the pose and request.